docs/docs/reference/enclaves.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Enclaves
+sidebar_label: Enclaves
+---
+
+An enclave is a Kurtosis primitive representing an isolated, ephemeral environment - like the Kurtosis version of a Kubernetes namespace. They are managed by the `enclave` family of CLI commands (e.g. [`kurtosis enclave add`][enclave-add-reference], [`kurtosis enclave ls`][enclave-ls-reference], [`kurtosis enclave inspect`][enclave-inspect-reference], etc.).
+
+Each enclave houses an arbitrary number of services and [files artifacts][files-artifacts-reference]. The contents of an enclave are manipulated via [Starlark instructions][starlark-instructions-reference] on [the `plan` object][plan-reference].
```

Comment:
Hm I don't think it was obvious to me before I read this that the CLI commands are *mostly* meant for managing enclave state while Starlark instructions are *mostly* meant to manipulate the contents of an enclave. 

If this is a design/usability tenant we want our users to adhere to, can I add a note to our CLI commands docs and our Starlark instructions docs? Kind of like a best practices with enclaves type of thing.